pyproject.toml

```diff
@@ -40,21 +40,33 @@ module = [
 ]
 ignore_missing_imports = true
 
-[tool.pycln]
-all = true
+[tool.ruff]
+# F: pyflakes
+# E, W: pycodestyles
+# where flake8 = pyflakes + pycodestyles
+# B: flake8 - bugbear
+# UP: pyupgrade
+# I: isort
+select = ['E', 'F', "W", 'B', 'UP', 'I']
```

Comment:
It might be better to follow normal bugbear usage and select `B0` + whatever extra B9 checks you use (`B904` and `B905` are the only ones there right now, IIRC). Unlike flake8, selecting "X" selects "X9*" too. Bugbear doesn't mind adding _very_ opinionated checks to the 9 series.

Reply:
Also, I recommend putting the comments inline, or they might get outdated. See https://github.com/scikit-build/scikit-build-core/blob/d64e68f7db91a1adee1d6d4ae281899a4f45b2a8/pyproject.toml#L192-L212 for example.